cloudfront/invalidation/lambda/src/cache_invalidation.ts

```diff
@@ -0,0 +1,62 @@
+import { SNSHandler, SNSMessage } from 'aws-lambda/trigger/sns';
```

Comment:
This file should be called `cacheInvalidation.ts` to keep in line with the other lambdas.